<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="ICDC_NewStudy" time="46.825" tests="4" failures="0" errors="4">
   <testsuite name="ICDC_NewStudy" tests="4" failures="0" errors="4" time="46.825" timestamp="2021-04-16 10:07:16" hostname="radhakrishnang2 - 10.242.31.140" id="Test Suites/ICDC_NewStudy">
      <properties>
         <property name="deviceName" value=""/>
         <property name="devicePlatform" value=""/>
         <property name="logFolder" value="C:\\Users\\radhakrishnang2\\Desktop\\Commons_Automation\\Reports\\20210416_100711\\ICDC_NewStudy\\20210416_100711"/>
         <property name="logFiles" value="C:\\Users\\radhakrishnang2\\Desktop\\Commons_Automation\\Reports\\20210416_100711\\ICDC_NewStudy\\20210416_100711\\execution0.log"/>
         <property name="attachments" value=""/>
         <property name="hostName" value="radhakrishnang2 - 10.242.31.140"/>
         <property name="os" value="Windows 10 64bit"/>
         <property name="katalonVersion" value="7.7.0.205"/>
         <property name="browser" value="Firefox 78.0"/>
         <property name="hostAddress" value="10.242.31.140"/>
         <property name="sessionId" value="b6ad6b50-f897-4bb2-aa2d-3bb349c9c761"/>
         <property name="seleniumVersion" value="3.141.59"/>
         <property name="proxyInformation" value="ProxyInformation { proxyOption=NO_PROXY, proxyServerType=HTTP, username=, password=********, proxyServerAddress=, proxyServerPort=0, executionList=&quot;&quot;, isApplyToDesiredCapabilities=true }"/>
         <property name="platform" value="Windows 10"/>
      </properties>
      <testcase name="Test Cases/Canine_TestCases/Study-Combinations/TC28_Canine_StudyUBC-AllBreeds_StageOfDisease" time="13.258" classname="Test Cases/Canine_TestCases/Study-Combinations/TC28_Canine_StudyUBC-AllBreeds_StageOfDisease" status="ERROR">
         <error type="ERROR" message="Test Cases/Canine_TestCases/Study-Combinations/TC28_Canine_StudyUBC-AllBreeds_StageOfDisease FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.invokeMethod(runtestcaseforKatalon.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.run(TC28_Canine_StudyUBC-AllBreeds_StageOfDisease:41)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:339)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:330)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:309)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:235)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:191)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:141)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:90)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1618592831483.run(TempTestSuite1618592831483.groovy:39)
Caused by: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.RunKatalon(runtestcaseforKatalon.groovy:75)
	... 16 more
"/>
         <system-out><![CDATA[2021-04-16 10:07:17 - [TEST_CASE][ERROR] - Test Cases/Canine_TestCases/Study-Combinations/TC28_Canine_StudyUBC-AllBreeds_StageOfDisease: Test Cases/Canine_TestCases/Study-Combinations/TC28_Canine_StudyUBC-AllBreeds_StageOfDisease FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.invokeMethod(runtestcaseforKatalon.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.run(TC28_Canine_StudyUBC-AllBreeds_StageOfDisease:41)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:339)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:330)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:309)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:235)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:191)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:141)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:90)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1618592831483.run(TempTestSuite1618592831483.groovy:39)
Caused by: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.RunKatalon(runtestcaseforKatalon.groovy:75)
	... 16 more


2021-04-16 10:07:18 - [TEST_STEP][PASSED] - openBrowser(""): Browser is opened with url: ''

2021-04-16 10:07:18 - [MESSAGE][INFO] - Starting 'Firefox (headless)' driver

2021-04-16 10:07:18 - [MESSAGE][INFO] - User set preference: ['args', '[--no-sandbox]']

2021-04-16 10:07:19 - [MESSAGE][INFO] - Action delay is set to 5,000 milliseconds

2021-04-16 10:07:28 - [MESSAGE][PASSED] - Browser is opened with url: ''

2021-04-16 10:07:28 - [TEST_STEP][PASSED] - maximizeWindow(): Current window maximized

2021-04-16 10:07:28 - [MESSAGE][PASSED] - Current window maximized

2021-04-16 10:07:28 - [TEST_STEP][PASSED] - "Driver opened by Katalon is used in Selenium from this step.": null

2021-04-16 10:07:28 - [TEST_STEP][ERROR] - ctdc.utilities.runtestcaseforKatalon.RunKatalon("TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx"): java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)

2021-04-16 10:07:28 - [MESSAGE][PASSED] - Test case file loaded This is the full filepath after converting to string :C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx

2021-04-16 10:07:30 - [MESSAGE][WARNING] - Test object id is null

2021-04-16 10:07:30 - [MESSAGE][INFO] - Global variable set for password file is :  C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx

2021-04-16 10:07:30 - [MESSAGE][ERROR] - java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)]]></system-out>
         <system-err><![CDATA[2021-04-16 10:07:17 - [TEST_CASE][ERROR] - Test Cases/Canine_TestCases/Study-Combinations/TC28_Canine_StudyUBC-AllBreeds_StageOfDisease: Test Cases/Canine_TestCases/Study-Combinations/TC28_Canine_StudyUBC-AllBreeds_StageOfDisease FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.invokeMethod(runtestcaseforKatalon.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.run(TC28_Canine_StudyUBC-AllBreeds_StageOfDisease:41)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:339)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:330)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:309)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:235)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:191)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:141)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:90)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1618592831483.run(TempTestSuite1618592831483.groovy:39)
Caused by: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.RunKatalon(runtestcaseforKatalon.groovy:75)
	... 16 more]]></system-err>
      </testcase>
      <testcase name="Test Cases/Canine_TestCases/Study-Combinations/TC29_Canine_StudyUBC-AllBreeds_StageOfDisease" time="8.578" classname="Test Cases/Canine_TestCases/Study-Combinations/TC29_Canine_StudyUBC-AllBreeds_StageOfDisease" status="ERROR">
         <error type="ERROR" message="Test Cases/Canine_TestCases/Study-Combinations/TC29_Canine_StudyUBC-AllBreeds_StageOfDisease FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC29_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.invokeMethod(runtestcaseforKatalon.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at TC29_Canine_StudyUBC-AllBreeds_StageOfDisease.run(TC29_Canine_StudyUBC-AllBreeds_StageOfDisease:41)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:339)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:330)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:309)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:235)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:191)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:141)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:90)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1618592831483.run(TempTestSuite1618592831483.groovy:39)
Caused by: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC29_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.RunKatalon(runtestcaseforKatalon.groovy:75)
	... 16 more
"/>
         <system-out><![CDATA[2021-04-16 10:07:30 - [TEST_CASE][ERROR] - Test Cases/Canine_TestCases/Study-Combinations/TC29_Canine_StudyUBC-AllBreeds_StageOfDisease: Test Cases/Canine_TestCases/Study-Combinations/TC29_Canine_StudyUBC-AllBreeds_StageOfDisease FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC29_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.invokeMethod(runtestcaseforKatalon.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at TC29_Canine_StudyUBC-AllBreeds_StageOfDisease.run(TC29_Canine_StudyUBC-AllBreeds_StageOfDisease:41)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:339)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:330)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:309)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:235)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:191)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:141)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:90)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1618592831483.run(TempTestSuite1618592831483.groovy:39)
Caused by: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC29_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.RunKatalon(runtestcaseforKatalon.groovy:75)
	... 16 more


2021-04-16 10:07:31 - [TEST_STEP][PASSED] - openBrowser(""): Browser is opened with url: ''

2021-04-16 10:07:31 - [MESSAGE][WARNING] - A browser is already opened. Closing browser and opening a new one

2021-04-16 10:07:33 - [MESSAGE][INFO] - Starting 'Firefox (headless)' driver

2021-04-16 10:07:33 - [MESSAGE][INFO] - User set preference: ['args', '[--no-sandbox]']

2021-04-16 10:07:33 - [MESSAGE][INFO] - Action delay is set to 5,000 milliseconds

2021-04-16 10:07:39 - [MESSAGE][PASSED] - Browser is opened with url: ''

2021-04-16 10:07:39 - [TEST_STEP][PASSED] - maximizeWindow(): Current window maximized

2021-04-16 10:07:39 - [MESSAGE][PASSED] - Current window maximized

2021-04-16 10:07:39 - [TEST_STEP][PASSED] - "Driver opened by Katalon is used in Selenium from this step.": null

2021-04-16 10:07:39 - [TEST_STEP][ERROR] - ctdc.utilities.runtestcaseforKatalon.RunKatalon("TC29_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx"): java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC29_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)

2021-04-16 10:07:39 - [MESSAGE][PASSED] - Test case file loaded This is the full filepath after converting to string :C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC29_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx

2021-04-16 10:07:39 - [MESSAGE][INFO] - Global variable set for password file is :  C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC29_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx

2021-04-16 10:07:39 - [MESSAGE][ERROR] - java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC29_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)]]></system-out>
         <system-err><![CDATA[2021-04-16 10:07:30 - [TEST_CASE][ERROR] - Test Cases/Canine_TestCases/Study-Combinations/TC29_Canine_StudyUBC-AllBreeds_StageOfDisease: Test Cases/Canine_TestCases/Study-Combinations/TC29_Canine_StudyUBC-AllBreeds_StageOfDisease FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC29_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.invokeMethod(runtestcaseforKatalon.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at TC29_Canine_StudyUBC-AllBreeds_StageOfDisease.run(TC29_Canine_StudyUBC-AllBreeds_StageOfDisease:41)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:339)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:330)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:309)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:235)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:191)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:141)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:90)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1618592831483.run(TempTestSuite1618592831483.groovy:39)
Caused by: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC29_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.RunKatalon(runtestcaseforKatalon.groovy:75)
	... 16 more]]></system-err>
      </testcase>
      <testcase name="Test Cases/Canine_TestCases/Study-Combinations/TC30_Canine_StudyUBC-AllBreeds_StageOfDisease" time="11.074" classname="Test Cases/Canine_TestCases/Study-Combinations/TC30_Canine_StudyUBC-AllBreeds_StageOfDisease" status="ERROR">
         <error type="ERROR" message="Test Cases/Canine_TestCases/Study-Combinations/TC30_Canine_StudyUBC-AllBreeds_StageOfDisease FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC30_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.invokeMethod(runtestcaseforKatalon.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at TC30_Canine_StudyUBC-AllBreeds_StageOfDisease.run(TC30_Canine_StudyUBC-AllBreeds_StageOfDisease:41)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:339)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:330)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:309)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:235)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:191)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:141)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:90)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1618592831483.run(TempTestSuite1618592831483.groovy:39)
Caused by: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC30_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.RunKatalon(runtestcaseforKatalon.groovy:75)
	... 16 more
"/>
         <system-out><![CDATA[2021-04-16 10:07:39 - [TEST_CASE][ERROR] - Test Cases/Canine_TestCases/Study-Combinations/TC30_Canine_StudyUBC-AllBreeds_StageOfDisease: Test Cases/Canine_TestCases/Study-Combinations/TC30_Canine_StudyUBC-AllBreeds_StageOfDisease FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC30_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.invokeMethod(runtestcaseforKatalon.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at TC30_Canine_StudyUBC-AllBreeds_StageOfDisease.run(TC30_Canine_StudyUBC-AllBreeds_StageOfDisease:41)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:339)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:330)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:309)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:235)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:191)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:141)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:90)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1618592831483.run(TempTestSuite1618592831483.groovy:39)
Caused by: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC30_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.RunKatalon(runtestcaseforKatalon.groovy:75)
	... 16 more


2021-04-16 10:07:40 - [TEST_STEP][PASSED] - openBrowser(""): Browser is opened with url: ''

2021-04-16 10:07:40 - [MESSAGE][WARNING] - A browser is already opened. Closing browser and opening a new one

2021-04-16 10:07:41 - [MESSAGE][INFO] - Starting 'Firefox (headless)' driver

2021-04-16 10:07:41 - [MESSAGE][INFO] - User set preference: ['args', '[--no-sandbox]']

2021-04-16 10:07:41 - [MESSAGE][INFO] - Action delay is set to 5,000 milliseconds

2021-04-16 10:07:50 - [MESSAGE][PASSED] - Browser is opened with url: ''

2021-04-16 10:07:50 - [TEST_STEP][PASSED] - maximizeWindow(): Current window maximized

2021-04-16 10:07:50 - [MESSAGE][PASSED] - Current window maximized

2021-04-16 10:07:50 - [TEST_STEP][PASSED] - "Driver opened by Katalon is used in Selenium from this step.": null

2021-04-16 10:07:50 - [TEST_STEP][ERROR] - ctdc.utilities.runtestcaseforKatalon.RunKatalon("TC30_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx"): java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC30_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)

2021-04-16 10:07:50 - [MESSAGE][PASSED] - Test case file loaded This is the full filepath after converting to string :C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC30_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx

2021-04-16 10:07:50 - [MESSAGE][INFO] - Global variable set for password file is :  C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC30_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx

2021-04-16 10:07:50 - [MESSAGE][ERROR] - java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC30_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)]]></system-out>
         <system-err><![CDATA[2021-04-16 10:07:39 - [TEST_CASE][ERROR] - Test Cases/Canine_TestCases/Study-Combinations/TC30_Canine_StudyUBC-AllBreeds_StageOfDisease: Test Cases/Canine_TestCases/Study-Combinations/TC30_Canine_StudyUBC-AllBreeds_StageOfDisease FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC30_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.invokeMethod(runtestcaseforKatalon.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at TC30_Canine_StudyUBC-AllBreeds_StageOfDisease.run(TC30_Canine_StudyUBC-AllBreeds_StageOfDisease:41)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:339)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:330)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:309)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:235)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:191)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:141)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:90)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1618592831483.run(TempTestSuite1618592831483.groovy:39)
Caused by: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC30_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.RunKatalon(runtestcaseforKatalon.groovy:75)
	... 16 more]]></system-err>
      </testcase>
      <testcase name="Test Cases/Canine_TestCases/Study-Combinations/TC31_Canine_StudyUBC-AllBreeds_StageOfDisease" time="11.176" classname="Test Cases/Canine_TestCases/Study-Combinations/TC31_Canine_StudyUBC-AllBreeds_StageOfDisease" status="ERROR">
         <error type="ERROR" message="Test Cases/Canine_TestCases/Study-Combinations/TC31_Canine_StudyUBC-AllBreeds_StageOfDisease FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC31_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.invokeMethod(runtestcaseforKatalon.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at TC31_Canine_StudyUBC-AllBreeds_StageOfDisease.run(TC31_Canine_StudyUBC-AllBreeds_StageOfDisease:41)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:339)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:330)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:309)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:235)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:191)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:141)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:90)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1618592831483.run(TempTestSuite1618592831483.groovy:39)
Caused by: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC31_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.RunKatalon(runtestcaseforKatalon.groovy:75)
	... 16 more
"/>
         <system-out><![CDATA[2021-04-16 10:07:50 - [TEST_CASE][ERROR] - Test Cases/Canine_TestCases/Study-Combinations/TC31_Canine_StudyUBC-AllBreeds_StageOfDisease: Test Cases/Canine_TestCases/Study-Combinations/TC31_Canine_StudyUBC-AllBreeds_StageOfDisease FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC31_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.invokeMethod(runtestcaseforKatalon.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at TC31_Canine_StudyUBC-AllBreeds_StageOfDisease.run(TC31_Canine_StudyUBC-AllBreeds_StageOfDisease:41)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:339)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:330)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:309)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:235)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:191)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:141)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:90)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1618592831483.run(TempTestSuite1618592831483.groovy:39)
Caused by: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC31_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.RunKatalon(runtestcaseforKatalon.groovy:75)
	... 16 more


2021-04-16 10:07:51 - [TEST_STEP][PASSED] - openBrowser(""): Browser is opened with url: ''

2021-04-16 10:07:51 - [MESSAGE][WARNING] - A browser is already opened. Closing browser and opening a new one

2021-04-16 10:07:53 - [MESSAGE][INFO] - Starting 'Firefox (headless)' driver

2021-04-16 10:07:53 - [MESSAGE][INFO] - User set preference: ['args', '[--no-sandbox]']

2021-04-16 10:07:53 - [MESSAGE][INFO] - Action delay is set to 5,000 milliseconds

2021-04-16 10:08:01 - [MESSAGE][PASSED] - Browser is opened with url: ''

2021-04-16 10:08:01 - [TEST_STEP][PASSED] - maximizeWindow(): Current window maximized

2021-04-16 10:08:01 - [MESSAGE][PASSED] - Current window maximized

2021-04-16 10:08:01 - [TEST_STEP][PASSED] - "Driver opened by Katalon is used in Selenium from this step.": null

2021-04-16 10:08:01 - [TEST_STEP][ERROR] - ctdc.utilities.runtestcaseforKatalon.RunKatalon("TC31_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx"): java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC31_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)

2021-04-16 10:08:01 - [MESSAGE][PASSED] - Test case file loaded This is the full filepath after converting to string :C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC31_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx

2021-04-16 10:08:01 - [MESSAGE][INFO] - Global variable set for password file is :  C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC31_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx

2021-04-16 10:08:01 - [MESSAGE][ERROR] - java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC31_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)]]></system-out>
         <system-err><![CDATA[2021-04-16 10:07:50 - [TEST_CASE][ERROR] - Test Cases/Canine_TestCases/Study-Combinations/TC31_Canine_StudyUBC-AllBreeds_StageOfDisease: Test Cases/Canine_TestCases/Study-Combinations/TC31_Canine_StudyUBC-AllBreeds_StageOfDisease FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC31_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.invokeMethod(runtestcaseforKatalon.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at TC31_Canine_StudyUBC-AllBreeds_StageOfDisease.run(TC31_Canine_StudyUBC-AllBreeds_StageOfDisease:41)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:339)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:330)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:309)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:235)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:191)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:141)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:90)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1618592831483.run(TempTestSuite1618592831483.groovy:39)
Caused by: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC31_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.RunKatalon(runtestcaseforKatalon.groovy:75)
	... 16 more]]></system-err>
      </testcase>
      <system-out><![CDATA[2021-04-16 10:07:16 - [TEST_SUITE][ERROR] - ICDC_NewStudy: Test Cases/Canine_TestCases/Study-Combinations/TC28_Canine_StudyUBC-AllBreeds_StageOfDisease FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.invokeMethod(runtestcaseforKatalon.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.run(TC28_Canine_StudyUBC-AllBreeds_StageOfDisease:41)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:339)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:330)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:309)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:235)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:191)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:141)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:90)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1618592831483.run(TempTestSuite1618592831483.groovy:39)
Caused by: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.RunKatalon(runtestcaseforKatalon.groovy:75)
	... 16 more]]></system-out>
      <system-err><![CDATA[2021-04-16 10:07:16 - [TEST_SUITE][ERROR] - ICDC_NewStudy: Test Cases/Canine_TestCases/Study-Combinations/TC28_Canine_StudyUBC-AllBreeds_StageOfDisease FAILED.
Reason:
org.codehaus.groovy.runtime.InvokerInvocationException: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.invokeMethod(runtestcaseforKatalon.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:50)
	at TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.run(TC28_Canine_StudyUBC-AllBreeds_StageOfDisease:41)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:339)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:330)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:309)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:301)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:235)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:191)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:141)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:90)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1618592831483.run(TempTestSuite1618592831483.groovy:39)
Caused by: java.io.FileNotFoundException: C:\Users\radhakrishnang2\Desktop\Commons_Automation\InputFiles\TC28_Canine_StudyUBC-AllBreeds_StageOfDisease.xlsx (The system cannot find the file specified)
	at ctdc.utilities.runtestcaseforKatalon.RunKatalon(runtestcaseforKatalon.groovy:75)
	... 16 more]]></system-err>
   </testsuite>
</testsuites>
